numpy/core/src/umath/loops.c.src

```diff
@@ -1644,6 +1644,7 @@ NPY_NO_EXPORT void
             const @type@ in1 = *(@type@ *)ip1;
             *((npy_bool *)op1) = @func@(in1) != 0;
         }
+        npy_clear_floatstatus();
```

Comment:
this needs to be one line lower, otherwise it will not cover the simd case. That function runs the code and only returns false if simd could not be used.